quick-start/src/main/ui/app/app.module.ts

```diff
@@ -42,7 +42,8 @@ import { SelectComponent } from './shared/components/select/select.component';
 import { SelectListComponent } from './shared/components/select-list/select-list.component';
 import { TooltipModule } from './tooltip';
 import { TracesComponent, TraceViewerComponent } from './traces';
-import { SearchComponent, SearchViewerComponent } from './search';
+import { SearchComponent,  } from './search';
+import { SearchViewerComponent } from './shared/components/search/search-viewer/search-viewer.component';
```

Comment:
Please create an entry in the index on this level "./shared/components" and use it as an import.